content/_includes/layout.macros.njk

```diff
@@ -25,7 +25,7 @@ params:
   content,
   align='right'
 ) %}
-<h2 data-section-title="{{ align }}">{{ content | mdInline | safe }}</h2>
+<h2 id="{{ content | slug }}" data-section-title="{{ align }}">{{ content | mdInline | safe }}</h2>
```

Comment:
@mirisuzanne I'm not 100% clear on how the `slug` works... but I think this is okay?

Reply:
Yeah, I'd expect that to work fine. At worst, it would sometimes be long?

Reply:
This could also be lowercased?

Reply:
The `slug` filter already makes it lowercase (https://github.com/11ty/eleventy/blob/master/src/Filters/Slug.js).